### PR TITLE
Doc: fix typos.

### PR DESCRIPTION
**Pull request by xylinkun**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

[Please explain **in detail** why the changes in this PR are needed.]

### CODE REVIEW

File: docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md [hunk 1]
```diff
@@ -31,18 +31,18 @@ SpaceVim 是一个模块化的 Vim IDE，针对 perl 这一语言的支持主要
...
-`lang#perl` 模块, 通过快捷键 `SPC f v d` 打开配置文件，添加：
+`lang#perl`模块，通过快捷键`SPC f v d`打开配置文件，添加：
 
 ```toml
 [[layers]]
   name = "lang#perl"
 ```
 
-启用 `lang#perl` 模块后，在打开 perl 文件时，就可以使用语言专属快捷键，这些快捷键都是以 `SPC l` 为前缀的。
+启用`lang#perl`模块后，在打开 Perl 文件时，就可以使用语言专属快捷键，这些快捷键都是以`SPC l`为前缀的。
```
Comment (wsdjeg) [inline docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md]: space arround `` `lang#perl` `` should not be removed

File: docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md [hunk 2]
```diff
@@ -51,8 +51,8 @@ SpaceVim 初次安装时默认并未启用相关语言模块。首先需要启
 
 ### 语法检查
 
-`checkers` 模块为 SpaceVim 提供了语法检查的功能，该模块默认已经载入。该模块默认使用 [neomake](https://github.com/neomake/neomake)
-这一异步语法检查工具。对于 perl 的支持，是通过异步调用 perl 和 perlcritic 命令来完成的。
+`checkers`模块为 SpaceVim 提供了语法检查的功能，该模块默认已经载入。该模块默认使用 [neomake](https://github.com/neomake/neomake)
+这一异步语法检查工具。对于 Perl 的支持，是通过异步调用 Perl 和 perlcritic 命令来完成的。
```
Comment (wsdjeg) [inline docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md]: 这里调用的是 perl 这个命令，不需要大写。

File: docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md (hunk 1 above)
Comment (xylinkun) [reply docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md]: It's ugly that there is a space between a Chinese charactor and `.

Comment (wsdjeg) [reply docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md]: 中文和英文代码，英文符号之间有空格是规范，网页上丑是因为网页的代码格式设置问题，我会来修改。

Comment (wsdjeg) [reply docs/_posts/2019-02-11-use-vim-as-a-perl-ide.md]: 删掉以下属性即可：
```
margin: 0 4px;
padding: 4px 6px;
```

**Pull request merged**